### PR TITLE
ENG-4965 implement manager and scenario step one

### PR DESCRIPTION
**Pull request by ichalagashvili**
[ENG-4965](https://entando.myjetbrains.com/youtrack/issue/ENG-4965/implement-business-logic-to-handle-API-call-and-pick-a-scenario) and [ENG-4964](https://entando.myjetbrains.com/youtrack/issue/ENG-4964/implement-scenario-1-no-external-references)

### CODE REVIEW

**Review by DMinnai — APPROVED** [review]

File: src/locales/it.js [hunk 1]
```diff
@@ -1610,5 +1611,6 @@ export default {
     'componentRepository.refreshBundleVersions': 'Aggiorna le versioni disponibili del bundle',
     'componentRepository.bundle.installVersionsRefreshed': 'Aggiornato',
     'componentRepository.hub.epcInstalledTip': 'Per aggiornare il menu EPC, aggiorna la pagina di AppBuilder nel tuo browser',
+    'ecr.componentUninstallError': 'Message error - Is not possible to uninstall the {name} component because it is used by the following components:',
```
Comment (DMinnai) [inline src/locales/it.js]: I think it should be written in italian

File: src/locales/pt.js [hunk 2]
```diff
@@ -1030,5 +1031,6 @@ export default {
     'componentRepository.refreshBundleVersions': 'Atualizar as versões disponíveis do pacote',
     'componentRepository.bundle.installVersionsRefreshed': 'Atualizado',
     'componentRepository.hub.epcInstalledTip': 'Para atualizar corretamente o menu EPC, atualize a página do AppBuilder em seu navegador',
+    'ecr.componentUninstallError': 'Message error - Is not possible to uninstall the {name} component because it is used by the following components:',
```
Comment (DMinnai) [inline src/locales/pt.js]: and this one in PT :)

**Review by DMinnai — CHANGES_REQUESTED** [review]

File: src/locales/it.js (hunk 1 above)
Comment (ichalagashvili) [reply src/locales/it.js]: done

File: src/locales/pt.js (hunk 2 above)
Comment (ichalagashvili) [reply src/locales/pt.js]: done

**Review by DMinnai — APPROVED** [review]

**Review by nicpuddu — APPROVED** [review]

**Pull request merged**